### PR TITLE
Create Octaviya-setup.yaml

### PR DESCRIPTION
**Pull request by slecker**

### CODE REVIEW

File: profiles/Octaviya-setup.yaml [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+- Light Thrown
+- Small Edged
+hunting_info:
+- :zone: rats
+  # :duration: 30
+  stop_on:
+  - Brawling
+  - Light Thrown
+  - Small Edged
+prehunt_buffs: 7958
```
Comment (rpherbig) [inline profiles/Octaviya-setup.yaml]: I don't think you want to buff here if you're hunting rats. I think that room can have leucros in it. Regardless, it's far away from the rats zone.

File: profiles/Octaviya-setup.yaml [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+- Outfitting
+- Scholarship
+- Outdoorsmanship
+- Mechanical Lore
+train_workorders:
+- Tailoring
+training_spells:
+  Augmentation:
+    abbrev: Centering
+    symbiosis: true
```
Comment (rpherbig) [inline profiles/Octaviya-setup.yaml]: This looks suspicious for a low circle Cleric. Do you have these spells and can use symbiosis?

Comment (slecker) [inline profiles/Octaviya-setup.yaml]: I have the spell but not symbiosis, sorry meant to take that out!

File: profiles/Octaviya-setup.yaml [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+    symbiosis: false
+#  Utility:
+#    abbrev: Uncurse
+#    symbiosis: true
+  Warding:
+    abbrev: MPP
+    symbiosis: false
+cyclic_training_spells:
+  Warding:
+    abbrev: MPP
```
Comment (rpherbig) [inline profiles/Octaviya-setup.yaml]: `MPP` is not a cyclic spell

Comment (slecker) [inline profiles/Octaviya-setup.yaml]: >.>

Comment (slecker) [inline profiles/Octaviya-setup.yaml]: Done!

**Pull request merged**